spec/apps/dummy/db/migrate/20210304014602_create_mock_users.rb

```diff
@@ -1,6 +1,7 @@
 class CreateMockUsers < ActiveRecord::Migration[6.1]
   def change
-    create_table :mock_users do |t|
+    create_table :mock_users, id: :uuid, primary_key: :primary_key do |t|
+      t.timestamps
```

Comment:
Since the migrations for the dummy test app are *only* used for the dummy test app, I decided to collapse them all down rather to the minimum. This means a later migration which added timestamps is now deleted in favour of just doing that here, where the table is created.